### PR TITLE
fix fluviewer log & error copying approach

### PR DESCRIPTION
**Pull request by jpalmer37**
Fixes: https://github.com/BCCDC-PHL/fluviewer-nf/issues/13

### CODE REVIEW

**Comment by jpalmer37** [comment]
Here's the new code structure: 
- `SAVE_LOGS` function will always run and rename appropriate logs / files in the `logs` folder of the work-dir
- `SAFE_EXIT` function will only run upon non-zero fluviewer error status and forcefully copy the `fluviewer*txt` files under `log` to the `outdir`
- under exit code 0, publishDir will capture the `fluviewer*txt` files

File: modules/fluviewer.nf [hunk 1]
```diff
@@ -130,12 +137,11 @@ process fluviewer {
     if [[ ! -f ${sample_id}_HA_consensus.fa ]]; then
         echo "HA segment consensus not generated. Skipping FindCleave.py..."
     else
-	python ${projectDir}/bin/FindCleave.py -i ${sample_id}_HA_consensus.fa -o ${sample_id}_HPAI.tsv
+	    python ${projectDir}/bin/FindCleave.py -i ${sample_id}_HA_consensus.fa -o ${sample_id}_HPAI.tsv
```
Comment (dfornika) [inline modules/fluviewer.nf:140]: I don't think this is a big problem but we shouldn't need to specify `${projectDir}/bin/FindCleave.py`. Nextflow should always put `${projectDir}/bin` on the `PATH` for each process, so (as long as the script is executable) you should just be able to say `FindCleave.py` and the system will know where to find it.

**Review by dfornika — DISMISSED** [review]
I just have the one optional suggestion. Let me know if you'd like to change that or just merge as-is. I'm approving either way.

File: modules/fluviewer.nf (hunk 1 above)
Comment (dfornika) [reply modules/fluviewer.nf]: I've updated this myself.

**Review by dfornika — APPROVED** [review]
Looks good!

**Pull request merged**